src/utils/helpers/formatNumbersToString.ts

```diff
@@ -1,5 +1,5 @@
 export function formatEmissions(emissions: number): string {
-    return emissions.toFixed(2) + " grams of CO2";
+    return emissions.toFixed(2);
```

Comment:
Would be good to store this number as a constant so it can be easily updated later

Reply:
The point of the whole function is to centralise the same 2 decimal precision for all emissions.
=> the decision is to not change the function